configure.ac

```diff
@@ -546,7 +546,7 @@ esac
 
 AC_ARG_WITH(iodbc,
 AS_HELP_STRING([--with-iodbc=DIR], [build odbc driver against iODBC in DIR]))
-if test "$with_iodbc"; then
+if test "x$with_iodbc" != "x" -a "x$with_iodbc" != "xno"; then
```

Comment:
Not a big fun of the x prefix. I would use just
`if test "$with_iodbc" != "" -a "$with_iodbc" != "no"; then`
same below